accio-main/src/main/java/io/accio/main/web/dto/DuckDBQueryDto.java

```diff
@@ -0,0 +1,113 @@
+package io.accio.main.web.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+public class DuckDBQueryDto
```

Comment:
This dto class is similar with `PreviewOutputDto`. Consider to merge them.

Reply:
Using `PreviewOutputDto` for API of DuckDB may be not appropriate. I don't have good idea for the new name of `PreviewOutputDto` for two API. Make a empty class `DuckDBQueryDto` extend `PreviewOutputDto` may not a good idea too. Could you help me give me a new name?

Reply:
Probability, just create a new one called `QueryRresultDto`? I think the format of `PreviewOutputDto` and `DucKDBQueryDto` are same. Both of them have `columns` and `data` (or called `rows`). Use one common DTO is enough.